### ISSUE TITLE
Broken necks aren't an instant death

### ISSUE THREAD

**Issue by blocker2225** [studio-minus/ppg-bugs-and-requests#1507]
**Feature description**
A human can break their neck (head). Usually this is fatal immediately, but in real life broken necks are actually treatable. The victim will die if left without medical help. If the head is mended, the human will stop dying, and will be able to be revived.
Once a human's neck broken, they are permanently ragdolled and require medical attention to resuscitate. Inspector will say "broken neck" if the head/entire human is inspected. Death comes within 12 seconds. A broken neck has a chance of being irreversible and will kill the human faster if it is.

**Feature purpose**
In real life, unless you do a 360 to someone's neck, they can still be revived with good medical attention. This is just a QoL feature.

**Additional context**
inspired by Criminality (roblox game), where if you break your neck the same happens

**Comment by DindinYT37** [comment]
If we go literally you're snapping his skull not his neck.

**Comment by blocker2225** [comment]
> If we go literally you're snapping his skull, not his neck.

but the sound is the sound of a neck cracking

**Comment by DindinYT37** [comment]
> > If we go literally you're snapping his skull, not his neck.
> 
> but the sound is the sound of a neck cracking

I know, I'm just saying if we were to go literally we would actually snap his skull, not his neck as you are right clicking his head and not his neck.

**Comment by Drogo-man** [comment]
> > > If we go literally you're snapping his skull, not his neck.
> > 
> > 
> > but the sound is the sound of a neck cracking
> 
> I know, I'm just saying if we were to go literally we would actually snap his skull, not his neck as you are right clicking his head and not his neck.

didn't I say the exact same thing and you told me it was the neck you broke? just saying. Or it was somebody else idk.

**Comment by DindinYT37** [comment]
> > > > If we go literally you're snapping his skull, not his neck.
> > > 
> > > 
> > > but the sound is the sound of a neck cracking
> > 
> > 
> > I know, I'm just saying if we were to go literally we would actually snap his skull, not his neck as you are right clicking his head and not his neck.
> 
> didn't I say the exact same thing and you told me it was the neck you broke? just saying. Or it was somebody else idk.

Yes, I was the one that told you it's meant to be the neck that's being broken when you break the "bone" of the head. But since you were right in a literal sense (as in we right clicked the head, not the neck), I decided to mention it here.
